### ISSUE TITLE
WaveCalibrator - "doxygen-like" rst docs should be copied/moved to proper doxygen docs (Trac #1296)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1290]
They'll be more likely to stay up to date this way

Migrated from https://code.icecube.wisc.edu/ticket/1296
```json
{
    "status": "closed", 
    "changetime": "2016-03-18T21:14:15", 
    "description": "They'll be more likely to stay up to date this way", 
    "reporter": "nega", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1458335655846260", 
    "component": "combo core", 
    "summary": "WaveCalibrator - \"doxygen-like\" rst docs should be copied/moved to proper doxygen docs", 
    "priority": "normal", 
    "keywords": "documentation", 
    "time": "2015-08-28T18:32:53", 
    "milestone": "", 
    "owner": "jbraun", 
    "type": "cleanup"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20151209T17:17:47: david.schultz changed milestone from "" to "February 2016"

**Comment by icecube-trac** [comment]
Trac update at 20160217T18:10:10: 
* jbraun changed status from "new" to "closed"
* jbraun commented:

> In changeset 142051:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="142051"
> Update docs to have a similar look/feel to other projects in offline-software.  Closes #1290.
> ```


* jbraun changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20160318T21:14:15: 
* olivas commented:

> Milestone February 2016 deleted


* olivas changed milestone from "February 2016" to ""
